data/adapters.fa

```diff
@@ -0,0 +1,308 @@
+>Reverse_adapter
```

Comment:
Is this something generic that can be used by many people or is specific to your own type of analysis? Wondering whether it should be e.g. in the testdata repository if its just useful for your testing / full test data if the latter. Otherwise is it somewhere documented what this file contains maybe with some sources?

Reply:
This file belongs to the BBDuK trimming tool. It contains a 'database' of adapters, which is provided with the tool. The path to this file is defined by one of its parameters https://github.com/nf-core/dualrnaseq/blob/769153c53efc7202f54a03c817e0926ca6985b5e/nextflow.config#L42

Reply:
Thanks for clarifying @bozmik ! Okay, please move this to `"${projectDir}/assets/bbduk-adapters.fa"` as we tend to keep special files like this one in the `assets` directory for all nf-core pipelines :-) This way its where other pipeline maintainers would expect it & its obvious and clear from which tool the file comes from and why you need to ship it with the actual pipeline - does that make sense? :-)

Reply:
yup - done